packages/mask/src/plugins/Tips/SNSAdaptor/bodyViews/AddWallet.tsx

```diff
@@ -0,0 +1,167 @@
+import { BindingProof, NextIDAction, NextIDPlatform } from '@masknet/shared-base'
+import { WalletMessages } from '@masknet/plugin-wallet'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import {
+    isSameAddress,
+    ProviderType,
+    resolveProviderName,
+    useAccount,
+    useChainId,
+    useNetworkType,
+    useProviderType,
+    useWallet,
+} from '@masknet/web3-shared-evm'
+import { memo, useCallback } from 'react'
+import { SignSteps, Steps } from '../../../../components/shared/VerifyWallet/Steps'
+import { useAsync, useAsyncFn } from 'react-use'
+import { NextIDProof } from '@masknet/web3-providers'
+import Services from '../../../../extension/service'
+import { useCustomSnackbar } from '@masknet/theme'
+import formatDateTime from 'date-fns/format'
+import { NetworkPluginID, useProviderDescriptor, useReverseAddress, useWeb3State } from '@masknet/plugin-infra/web3'
+import { useI18N } from '../../../../utils'
+
+interface AddWalletViewProps {
+    currentPersona: any
+    bindings: BindingProof[]
+    onCancel: () => void
+}
+
+const AddWalletView = memo(({ currentPersona, bindings, onCancel }: AddWalletViewProps) => {
+    const { t } = useI18N()
+    const { showSnackbar } = useCustomSnackbar()
+    const providerType = useProviderType()
+    const wallet = {
+        ...useWallet(),
+        account: useAccount(),
+        networkType: useNetworkType(),
+        chainId: useChainId(),
+        providerType,
+    }
+    const { value: domain } = useReverseAddress(wallet.account)
+    const { Utils } = useWeb3State() ?? {}
+    const isNotEvm = useProviderDescriptor()?.providerAdaptorPluginID !== NetworkPluginID.PLUGIN_EVM
+    const nowTime = formatDateTime(new Date(), 'yyyy-MM-dd HH:mm')
+    const isBound = bindings.filter((x) => isSameAddress(x.identity, wallet.account)).length > 0
```

Comment:
```suggestion
    const isBound = bindings.some((x) => isSameAddress(x.identity, wallet.account))
```

Reply:
[done in]https://github.com/DimensionDev/Maskbook/pull/6121